lib/services/iothub/README.md

```diff
@@ -0,0 +1,264 @@
+# Microsoft Azure SDK for Node.js - IoT Hub
+
+This project provides a Node.js package that makes it easy to manage Microsoft Azure Iot Hub Resources.Right now it supports:
+- **Node.js version: 4.x.x or higher**
+
+## How to Install
+
+```bash
+npm install azure-arm-iothub
+```
+## How to Use
+
+### Authentication, client creation 
+
+ ```javascript
+ var msrestAzure = require('ms-rest-azure');
+ var IoTHubClient = require('azure-arm-iothub');
+ 
+ // Create the IoTHub client
+            client = new IoTHubClient(suite.credentials, suite.subscriptionId, null);
```

Comment:
This code snippet in the readme is not usable by the customer. They have no idea what the suite is.

For writing scripts using the sdk, we have created auth api.

msrestAzure.interactiveLogin().

Check the sample in the [arm storagemanagement readme](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/storageManagement2#authentication-client-creation-and-listing-storageaccounts-as-an-example).